inference-engine/src/mkldnn_plugin/mkldnn_node.h

```diff
@@ -197,6 +197,106 @@ static std::string NameFromType(Type type) {
     }
 }
 
+static std::string algToString(Algorithm alg) {
```

Comment:
Move to separate file together with NameFromType

Reply:
Done
Has to keep static, otherwise linkage is failed.
As alternative we could create cpu_types.cpp

Reply:
Lets create cpu_types.cpp

Reply:
Done